client/state/data-layer/wpcom/read/following/mine/index.js

```diff
@@ -19,6 +19,13 @@ import { errorNotice } from 'state/notices/actions';
 const ITEMS_PER_PAGE = 200;
 const MAX_ITEMS = 2000;
 
+function toNumberOrUndefined( val ) {
+	if ( val ) {
+		return Number( val );
```

Comment:
this could come back as `0` or `NaN`. should we catch those and turn them into `undefined` too?